lecture-10/README.md

```diff
@@ -0,0 +1,7 @@
+# Lecture 10
+
+git clone ...
```

Comment:
If think we should add the url here @nyanhp

Reply:
🥳 Sure!

Reply:
got it!